kroxylicious-docs/docs/_snippets/snip-subject-builder-common-params.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: SNIPPET
+
+where:
+
+* <field name> identifies the certificate field to extract. Valid field names are listed in the **Supported fields** table.
```

Comment:
in the SASL inspector guide, it's talking about `certificate` which looks incorrect. maybe we can just drop the `certificate` term from the sentence.